### DISCUSSION TITLE
How to Reset the Dynamic Default Admin Password

### DISCUSSION THREAD

**Discussion by cocova** [mediacms-io/mediacms#325] [Q&A] [answered]
I finally get everything run well, but I forget the default passwd of admin.
Where could I RESET the admin password, or how to get it?

Thanks in advance.

**Comment by cocova** [comment, marked as answer]
The problem resolved by drop the database mediacms and drop the user mediacms in postgres.

**Comment by vikasgupta1812** [comment]
Hi @cocova - You can also reset the password by following below steps. Last command will prompt you to enter the desired password. 

```sh
cd /home/mediacms.io/mediacms
source  /home/mediacms.io/bin/activate
python manage.py changepassword admin
```
